python/src/viewer.py

```diff
@@ -427,6 +429,37 @@ def __init__(self,
                             self._ax[1 + i * dim + j].text(
                                 0.5, 0.5, labels[i], **text_kwargs)
 
+            elif drawableKind == 'Text':
+                dim = drawable.getData().getDimension()
+
+                # adjust font
+                if ('fontsize' not in text_kwargs_default) and ('size' not in text_kwargs_default):
+                    text_kwargs['fontsize'] = max(16 - dim, 4)
+                if 'horizontalalignment' in text_kwargs_default:
+                    horizontal_default = text_kwargs['horizontalalignment']
+                    del text_kwargs['horizontalalignment']
+                else:
+                    horizontal_default = 'center'
+                if 'verticalalignment' in text_kwargs_default:
+                    vertical_default = text_kwargs['verticalalignment']
+                    del text_kwargs['verticalalignment']
+                else:
+                    vertical_default = 'center'
+                positions = drawable.getTextPositions()
```

Comment:
positions is now a description, seems you forgot to change that part of the code